onset_maproom/calc.py

```diff
@@ -2,14 +2,16 @@
 import pandas as pd
 import xarray as xr
 
+#Date Reading functions
+
 def read_zarr_data(zarr_path):
   zarr_data = xr.open_zarr(zarr_path)
```

Comment:
switch to 4-space indent

Reply:
I still see a lot of lines that start with six spaces instead of eight.

What editor are you using? It probably has a setting to do this automatically. Black will also do it for you. You'll have to install it in your conda environment first, with `conda install -c conda-forge black`.

Reply:
balcked